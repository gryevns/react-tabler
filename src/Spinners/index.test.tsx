import React from "react";
import { shallow } from "enzyme";

import Spinner from "./index";

describe("Spinners", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<Spinner color="red" size="sm" grow />);
        expect(wrapper).toMatchSnapshot();
    });
});
