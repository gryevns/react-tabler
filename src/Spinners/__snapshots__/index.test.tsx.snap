// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinners should match snapshot 1`] = `
<div
  className="spinner-grow text-red spinner-border-sm"
  role="status"
/>
`;
